lib/recurly/errors.js

```diff
@@ -272,13 +272,13 @@ function recurlyErrorFactory (definition) {
       this.message += ` (need help? ${this.help})`;
     }
 
-    if (this.reporter instanceof EventDispatcher) {
+    if (this.reporter instanceof Reporter) {
       let type = 'client';
       if (this.classification) type += `:${this.classification}`;
 
       // Warning: any errors that occur in this code path risk
       // a stack overflow if they include a reporter
```

Comment:
What is done to prevent said stack overflow?

Reply:
Not including a reporter call.